config/boards/coolpi-cm5.csc

```diff
@@ -0,0 +1,40 @@
+# Rockchip RK3588 SoC octa core 4-16GB SoC 2.5GBe PoE eMMC USB3 NVME
+BOARD_NAME="CoolPi CM5"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER="Andy Yan"
+BOARD_FIRMWARE_INSTALL="-full"
+BOOT_SOC="rk3588"
+BOOTCONFIG="coolpi-cm5-evb-rk3588_defconfig"
+KERNEL_TARGET="edge"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-coolpi-cm5-evb.dtb"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOT_SPI_RKSPI_LOADER="yes"
+IMAGE_PARTITION_TABLE="gpt"
+BOOTFS_TYPE="ext4"
+
+# Mainline U-Boot
+function post_family_config_branch_edge__rock-5b_use_mainline_uboot() {
```

Comment:
It's better to change rock-5b to board name.

Reply:
Thanks for catching this， I will fix it in the next version。